### DISCUSSION TITLE
데이터 버퍼/캐시의 데이터는 어떻게 set 될까?

### DISCUSSION THREAD

**Discussion by nahyunLee** [Well-Founded-Confidence/RealMySQL#24] [Q&A]
4.1.2.1 포그라운드 스레드(클라이언트 스레드) 장의 내용입니다 (p.82)

> '포그라운드 스레드는 데이터를 MySQL의 데이터 버퍼나 캐시로부터 가져오며, 버퍼나 캐시에 없는 경우에는 직접 디스크의 데이터나 인덱스 파일로부터 데이터를 읽어와서 작업을 처리한다'

전반적으로 버퍼/캐시에서 데이터를 get해오는 내용은 있으나, set하는 정보는 없는 것 같네요...
miss날 때 직접 디스크에서 가져온 데이터를 바로 버퍼/캐시에 set하는 구조일까요?🤔

**Comment by menthamin** [comment]
Set, Expire 전략이 있을 것이다.. (뒷장에서)
- LRU (Least Recently Used)
